example/pubspec.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   proj4dart: ^2.1.0
   shared_preferences: ^2.3.2
   url_launcher: ^6.3.0
+  url_launcher_android: 6.3.2
```

Comment:
Do we need to depend on this directly @monsieurtanuki?

Reply:
I guess not, indeed. I think I was trapped in a WTF situation that was somehow solved later by #1985.